modules/run-pex-as-resource/main.tf

```diff
@@ -18,7 +18,24 @@ module "pex_env" {
 resource "null_resource" "run_pex" {
   count = var.enabled ? 1 : 0
 
-  triggers = var.triggers
+  triggers = merge(
+    var.triggers != null ? var.triggers : {},
+
+    # We store various values in the triggers so that they can be used in the destroy time provisioner.
+    # See https://github.com/hashicorp/terraform/issues/23679 for more details.
+    {
+      __pythonpath = module.pex_env.python_path
+      __env        = jsonencode(var.env)
+      __destroy_provisioner_command_args = (
+        var.enable_destroy_provisioner
+        # NOTE: The nested string interpolation can not be extracted because of the reference to self.
+        ? "${local.python_call} ${var.destroy_command_args}"
+        : "echo 'Skipping delete provisioner'"
```

Comment:
This doesn't seem to quite match what was in the original `command` of the `destroy` provisioner... Is that intentional?